test/e2e/set/scylladbcluster/multidatacenter/scylladbcluster_globalmanager.go

```diff
@@ -0,0 +1,211 @@
+		ns, nsClient := f.CreateUserNamespace(ctx)
+
+		workerClusters := f.WorkerClusters()
+		o.Expect(workerClusters).NotTo(o.BeEmpty(), "At least 1 worker cluster is required")
+
+		rkcMap, rkcClusterMap, err := utils.SetUpRemoteKubernetesClusters(ctx, f, workerClusters)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By(`Creating a ScyllaDBCluster with the global ScyllaDB Manager registration label`)
+		sc := f.GetDefaultScyllaDBCluster(rkcMap)
+		metav1.SetMetaDataLabel(&sc.ObjectMeta, naming.GlobalScyllaDBManagerRegistrationLabel, naming.LabelValueTrue)
+
+		sc, err = f.ScyllaAdminClient().ScyllaV1alpha1().ScyllaDBClusters(ns.Name).Create(ctx, sc, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = utils.RegisterCollectionOfRemoteScyllaDBClusterNamespaces(ctx, sc, rkcClusterMap)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the ScyllaDBCluster %q roll out (RV=%s)", sc.Name, sc.ResourceVersion)
+		rolloutCtx, rolloutCtxCancel := utils.ContextForMultiDatacenterScyllaDBClusterRollout(ctx, sc)
+		defer rolloutCtxCancel()
+		sc, err = controllerhelpers.WaitForScyllaDBClusterState(rolloutCtx, f.ScyllaAdminClient().ScyllaV1alpha1().ScyllaDBClusters(sc.Namespace), sc.Name, controllerhelpers.WaitForStateOptions{}, utils.IsScyllaDBClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		scylladbclusterverification.Verify(ctx, sc, rkcClusterMap)
+		err = scylladbclusterverification.WaitForFullQuorum(ctx, rkcClusterMap, sc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		smcrName, err := naming.ScyllaDBManagerClusterRegistrationNameForScyllaDBCluster(sc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for ScyllaDBCluster to register with global ScyllaDB Manager instance")
+		registrationCtx, registrationCtxCancel := context.WithTimeout(ctx, utils.SyncTimeout)
+		defer registrationCtxCancel()
+		smcr, err := controllerhelpers.WaitForScyllaDBManagerClusterRegistrationState(
+			registrationCtx,
+			nsClient.ScyllaClient().ScyllaV1alpha1().ScyllaDBManagerClusterRegistrations(ns.Name),
+			smcrName,
+			controllerhelpers.WaitForStateOptions{},
+			utilsv1alpha1.IsScyllaDBManagerClusterRegistrationRolledOut,
+			hasDeletionFinalizer,
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(smcr.Status.ClusterID).NotTo(o.BeNil())
+		o.Expect(*smcr.Status.ClusterID).NotTo(o.BeEmpty())
+		managerClusterID := *smcr.Status.ClusterID
+
+		managerClient, err := utils.GetManagerClient(ctx, f.KubeAdminClient().CoreV1())
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Verifying that ScyllaDBCluster was registered with global ScyllaDB Manager")
+		managerCluster, err := managerClient.GetCluster(ctx, managerClusterID)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(managerCluster.Labels).NotTo(o.BeNil())
+		o.Expect(managerCluster.Labels[naming.OwnerUIDLabel]).To(o.Equal(string(smcr.UID)))
```

Comment:
Wouldn't it make sense to extract a local helper that would prepare a labeled SDBC (e.g., `prepareScyllaDBClusterRegisteredWithManager`)? There are quite a few steps to perform, and they're all the same for both new test cases. Converting this into a table test (with a pluggable fn that should trigger deregistration) might be an option as well. WDYT?

Reply:
changed to DescribeTable as agreed internally